test/contractions.jl

```diff
@@ -62,9 +62,10 @@ end
         end
 
         sparse_result = TensorOperations.ncon(tensors, indices, conjlist)
-        dense_result = TensorOperations.ncon(Array.(tensors), indices, conjlist)
-
-        @test Array(sparse_result) ≈ dense_result
+        if SparseArrayKit.nonzero_length(sparse_result) > 0
```

Comment:
Does this not work if the result is completely sparse?

Reply:
It does, but the tests take quite some time, my impression being that it is the dense tensor contraction taking most of the time. And since quite a few of the tests actually yield a zero tensor, I thought to not contract these in the dense case to save on compute time. The resulting test is still meaningful, but I would hope that the tests leading to nonzero tensors would be sufficient to check for errors.

Reply:
Maybe just a comment for future reference would be helpful then :)